tests/functional/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```
@@ -631,6 +634,37 @@ def test_outcluster_nfs_export(
 
+        # Verify df on the external client reports PVC size, not cluster capacity
+        df_cmd = f"df -h {self.test_folder}"
+        retcode, stdout, stderr = self.con.exec_cmd(df_cmd)
+        assert retcode == 0, f"df command failed: {stderr}"
+        log.info("df -h output on external client:\n%s", stdout)
+        # df -h output line format:
+        #   Filesystem  Size  Used  Avail  Use%  Mounted on
+        #   server:/path  5.0G  4.0M  5.0G  1%  /mnt/test
+        df_line = [
+            line for line in stdout.strip().splitlines() if self.test_folder in line
+        ][0]
```

Ruff (0.15.17)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)